index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="ochieng.css">
+  </head>
+  <body>
+    <!-- navbar -->
+    <nav id="navbar">
+      <a style="padding: 10px;" href="#welcome-section">Welcome</a>
+      <a style="padding: 10px;" href="#projects">Projects</a>
+      <a style="padding: 10px;" id="profile-link" href="#" target="_blank">My GitHub Profile</a>
```

Comment:
Please consider using `<ul>` for listing the items on your navbar. Using a list element for a list of items is more semantically correct than using a series